src/addrman.h

```diff
@@ -146,11 +146,14 @@ class AddrMan
     /**
      * Choose an address to connect to.
      *
-     * @param[in] newOnly  Whether to only select addresses from the new table.
+     * @param[in] new_only Whether to only select addresses from the new table. Passing `true` returns
+     *                     an address from the new table or an empty pair. Passing `false` will return an
+     *                     address from either the new or tried table (it does not guarantee a tried entry).
+     * @param[in] network  Select only addresses of this network (nullopt = all)
```

Comment:
Probably worth documenting that passing in a `network` here rather than `nullopt` can make the search significantly slower (though still fast)?